src/test/resources/cucumber/features/integrationTests/cli/astCliScan.feature

```diff
@@ -0,0 +1,39 @@
+@AST_CLI_SCAN  @IntegrationTest
+Feature: AST support in CxFlow command-line
+
+    @Skip
+    Scenario Outline: AST CLI scan of a local directory
+        Given scanner is <scanner>
+        And source directory contains vulnerable files
+        When running CxFlow with `scan local sources` options
+        Then bug tracker contains <issueNumber> issues
+        And no exception is thrown
+        Examples:
+            | scanner | issueNumber |
+            #| SCA     | 5           |
+            | AST     | 32          |
```

Comment:
Please be aware that tests with fixed issue counts may be fragile. I.e. issue count may change due to changes on the  backend. I've already encountered it with SCA in astScanProcessing.feature.

Reply:
I will remove the SCA here

Reply:
use a description of what is the meaning of the number instead of the actual number